src/it/spring-fat/pom.xml

```diff
@@ -109,6 +117,16 @@ SOFTWARE.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
@h1alexbel Does it necessary? Can this test work without specifying `-parameters` argument?